hashes-generator.php

```diff
@@ -4,42 +4,45 @@
  Author: mattyrob
  Author URI: http://subscribe2.wordpress.com/
 */
-$latest = 'http://wordpress.org/latest.zip';
-$file = 'latest.zip';
+$latest_url = 'http://wordpress.org/latest.zip';
+$zips = array_merge( glob( 'latest.zip' ), glob( 'wordpress-*.zip' ) );
```

Comment:
Using a wildcard in `glob()` is kinda dangerous, we should have a more stringent regex instead